roles/kubernetes-apps/ansible/tasks/nodelocaldns.yml

```diff
@@ -9,10 +22,32 @@
     - { name: nodelocaldns, file: nodelocaldns-deamonset.yml, type: daemonset }
   register: nodelocaldns_manifests
   vars:
-    clusterIP: "{{ skydns_server }}"
-    secondaryclusterIP: "{{ skydns_server_secondary }}"
+    forwardTarget: >-
+      {%- if secondaryclusterIP is defined and dns_mode == 'coredns_dual' -%}
+      {{ clusterIP }} {{ secondaryclusterIP }}
+      {%- else -%}
+      {{ clusterIP }}
+      {%- endif -%}
+    upstreamForwardTarget: >-
+      {%- if resolvconf_mode == 'host_resolvconf' and upstream_dns_servers is defined and upstream_dns_servers|length > 0 -%}
+      {{ upstream_dns_servers|join(' ') }}
+      {%- else -%}
+      /etc/resolv.conf
+      {%- endif -%}
   when:
     - enable_nodelocaldns == True
     - inventory_hostname == groups['kube-master'] | first
   tags:
     - nodelocaldns
+
+- name: Debug | Get content of nodelocaldns-config
+  slurp:
+    src: "{{ kube_config_dir }}/nodelocaldns-config.yml"
+  register: nodelocaldns_config
+  when:
+    - enable_nodelocaldns == True
+    - inventory_hostname == groups['kube-master'] | first
+
+- name: Debug templates
+  debug:
+    msg: "{{ nodelocaldns_config }}"
```

Comment:
I think this should be `nodelocaldns_config ['content'] | b64decode` shouldn't it?

Reply:
I think this can be omitted all together no?